model/resource/deployment_environment_name.yaml

```diff
@@ -4,5 +4,5 @@ groups:
     brief: >
       The software deployment.
     attributes:
-      - ref: deployment.environment
+      - ref: deployment.environment.name
```

Comment:
why you have renamed file as well? the file is not about particular attribute but the whole resource which deployment

Reply:
I'm not sure how it got renamed TBH. Might've happened when I was swapping between computers to fix some of the local workflows. Probably made a mistake. Thanks for the catch!